### PR TITLE
SOC-579 Fix strict warning for extra function param

### PR DESCRIPTION
**Pull request by mklucsarits**
https://wikia-inc.atlassian.net/browse/SOC-579

This change fixes a strict standards warning:
`Strict standards: Declaration of WikiaSQLCache::generateKey() should be compatible with FluentSql\Cache\Cache::generateKey(FluentSql\Breakdown $breakDown) in /usr/wikia/source/app/includes/wikia/WikiaSQLCache.class.php on line 10`

I've borrowed the fix made by @nmonterroso here, https://github.com/Wikia/app/commit/5a2919411f72541ab5bd9b2fd1970e7f7e4e7c81, which was later reverted.

### CODE REVIEW

**Comment by mklucsarits** [comment]
@nmonterroso @armonr Please have a look.

File: includes/wikia/WikiaSQLCache.class.php [hunk 1]
```diff
@@ -8,12 +8,21 @@
  */
 
 class WikiaSQLCache extends FluentSql\Cache\Cache {
+	/** @var bool */
+	private $useSharedKey = false;
```
Comment (armonr) [inline includes/wikia/WikiaSQLCache.class.php]: This is being initialized in the constructor, so no need to initialize here.

**Comment by armonr** [comment]
Other than the one comment, LGTM

**Comment by armonr** [comment]
:+1:

**Comment by nmonterroso** [comment]
LGTM

**Pull request merged**